scipy/signal/tests/test_windows.py

```diff
@@ -92,6 +93,75 @@ def test_basic(self):
                          6.0e-05])
 
 
+class TestTaylor(object):
+
+    def test_normalized(self):
+        """Tests windows of small length that are normalized to 1. See the
+        documentation for the Taylor window for more information on
+        normalization.
+        """
+        assert_allclose(windows.taylor(1, 2, 15), 1.0)
+        assert_allclose(
+            windows.taylor(5, 2, 15),
+            np.array([0.75803341, 0.90757699, 1.0, 0.90757699, 0.75803341])
+        )
+        assert_allclose(
+            windows.taylor(6, 2, 15),
+            np.array([
+                0.7504082, 0.86624416, 0.98208011, 0.98208011, 0.86624416, 0.7504082
+            ])
+        )
+
+    def test_non_normalized(self):
+        """Test windows of small length that are not normalized to 1. See the
+        documentation for the Taylor window for more information on
+        normalization.
+        """
+        assert_allclose(
+            windows.taylor(5, 2, 15, norm=False),
+            np.array([0.87508054, 1.04771499, 1.15440894, 1.04771499, 0.87508054])
+        )
+        assert_allclose(
+            windows.taylor(6, 2, 15, norm=False),
+            np.array([0.86627793, 1.0, 1.13372207, 1.13372207, 1.0, 0.86627793])
+        )
+
+    def test_correctness(self):
+        """This test ensures the correctness of the implemented Taylor
+        Windowing function. A Taylor Window of 1024 points is created, its FFT
+        is taken, and the Peak Sidelobe Level (PSLL) and 3dB and 18dB bandwidth
+        are found and checked.
+
+        A publication from Sandia National Laboratories was used as reference
+        for the correctness values [1]_.
+
+        References
+        -----
+        .. [1] Armin Doerry, "Catalog of Window Taper Functions for
+               Sidelobe Control", 2017.
+               https://www.researchgate.net/profile/Armin_Doerry/publication/316281181_Catalog_of_Window_Taper_Functions_for_Sidelobe_Control/links/58f92cb2a6fdccb121c9d54d/Catalog-of-Window-Taper-Functions-for-Sidelobe-Control.pdf
+        """
+        M_win = 1024
+        N_fft = 131072
+        # Set norm=False for correctness as the values obtained from the scientific
+        # publication do not normalize the values. Normalizing changes the sidelobe
+        # level from the desired value.
```

Comment:
https://travis-ci.org/github/scipy/scipy/jobs/741491067#L443-L449
```
./scipy/signal/tests/test_windows.py:111:80: E501 line too long (84 > 79 characters)
./scipy/signal/tests/test_windows.py:122:80: E501 line too long (82 > 79 characters)
./scipy/signal/tests/test_windows.py:126:80: E501 line too long (80 > 79 characters)
./scipy/signal/tests/test_windows.py:146:80: E501 line too long (83 > 79 characters)
./scipy/signal/tests/test_windows.py:147:80: E501 line too long (83 > 79 characters)
./scipy/signal/windows/windows.py:1702:80: E501 line too long (80 > 79 characters)
6       E501 line too long (84 > 79 characters)
```

Reply:
The last line, `6 ... line too long` doesn't point to a file, so I'm not sure where to look.

Reply:
Oh I see, it's just a total of the number E501 line too long errors displaying the line with the longest number of characters. These should all be fixed in the latest commit.